xonsh/completer.py

```diff
@@ -136,10 +138,19 @@ def cmd_complete(self, cmd):
         return {s + space for s in self._all_commands() if s.startswith(cmd)}
 
     def module_complete(self, prefix):
-        """Completes a name of a module to import"""
+        """Completes a name of a module to import."""
         modules = set(sys.modules.keys())
         return {s for s in modules if s.startswith(prefix)}
 
+    def option_complete(self, prefix, cmd):
+        """Completes an option name, basing on content of man page."""
+        manpage = subprocess.Popen(["man", cmd], stdout=subprocess.PIPE)
+        # This is a trick to get rid of reverse line feeds
+        text = subprocess.check_output(["col", "-b"], stdin=manpage.stdout).decode('utf-8')
```

Comment:
Well, actually I was looking for such an option in man manual. But I'm developing under OS X and here I have man version 1.6c (which was preinstalled) and there is no such option.
So I believe this version with pipes is going to work on greater number of machines. Maybe there's some other clean & nice solution?